frontend/src/components/shareContent/SocialMediaShareDialog.js

```diff
@@ -0,0 +1,136 @@
+import LinkIcon from "@material-ui/icons/Link";
+import GenericDialog from "../dialogs/GenericDialog";
+import React from "react";
+import { Button, InputAdornment, makeStyles, TextField } from "@material-ui/core";
+import {
+  EmailShareButton,
+  FacebookShareButton,
+  TwitterShareButton,
+  WhatsappShareButton,
+  LinkedinShareButton,
+  RedditShareButton,
+  TelegramShareButton,
+  EmailIcon,
+  FacebookIcon,
+  TwitterIcon,
+  WhatsappIcon,
+  LinkedinIcon,
+  RedditIcon,
+  TelegramIcon,
+} from "react-share";
+
+const useStyles = makeStyles((theme) => ({
+  shareButtonsContainer: {
+    paddingBottom: theme.spacing(2),
+    display: "flex",
+    justifyContent: "space-between",
+    [theme.breakpoints.down("xs")]: {
+      justifyContent: "flex-start",
+      flexWrap: "wrap",
+    },
+  },
+}));
+
+export default function SocialMediaShareDialog({
+  open,
+  onClose,
+  texts,
+  createShareRecord,
+  screenSize,
+  SHARE_OPTIONS,
+  projectLink,
+  title,
+}) {
+  const classes = useStyles();
+
+  const handleClose = () => {
+    onClose(false);
+  };
+
+  const facebookHashtag = "#believeintogether";
+  const twitterHastags = ["believeintogether"];
```

Comment:
I'd use camel case of hte hashtags (BelieveInTogether)

Reply:
done :+1: